Junk4Dollars-app/Auctions/AuctionDetailsViewController.swift

```diff
@@ -32,7 +32,14 @@ public class AuctionDetailsViewController: UIViewController, AuctionDetailsView
             self.auctionTitleLabel?.text = auction.title
             self.auctionDescriptionLabel?.text = auction.description
 
-            let price = Price(inCents: auction.starting_price).inDollars()
+            var price: String
+
+            if let bid = auction.bid {
+                price = Price(inCents: bid.price).inDollars()
```

Comment:
Cool thing Swift syntax allows:
```suggestion
                price = Price(in cents: bid.price).inDollars()
```

Reply:
Yeah, being able to have a separate argument label vs argument name is pretty cool. Do you think it's a good idea to change it here? If I commit this suggestion, instead of reading as 'price in cents(100)', using this method would read as just 'price in (100)'. I feel like the current way is more readable and helps follow the scheme I'm using with the `inDollars` method to convert prices.